### ISSUE TITLE
update 'run' parameter

### ISSUE THREAD

**Issue by senthil10** [NationalGenomicsInfrastructure/TACA#77]
As of now `--run` accepts only path, could be useful that it can be a file as well (with list of runs to archive).

@guillermo-carrasco : Would it be a over kill ?

**Comment by guillermo-carrasco** [comment]
It does not accept only a path, you can specify only a filename, and then it will prepend the `storage/archive_dirs` directory specified in the configuration, have a look [here](https://github.com/SciLifeLab/TACA/blob/master/taca/storage/storage.py#L54).

**Comment by senthil10** [comment]
Yes, I am aware of that :) When I meant `PATH` I really meant like we can give only one run for example like 

```taca storage --days 2 --run 150224_D00134_0180_AC6AHVANXX archive --backend swestore```

But what this issue is about to have some thing like this

```taca storage --days 2 --run runs_to_archive.txt archive --backend swestore```

```
cat runs_to_archive.txt
150202_D00134_0173_BHB6WVADXX
150204_D00134_0174_AHB6H1ADXX
```

So would that be useful ? or an over kill ? :)

**Comment by guillermo-carrasco** [comment]
aha I see, no, it may be useful and you're not adding extra options, which is nice. Just have to add the piece of code that determines if it's a text file and read it if so.

Go for it!

Just, don't forget to document all of this somehow, in the package docs for example, or it will be forgotten soon!
